src/proxy/ClientTable.php

```diff
@@ -107,65 +107,108 @@ public function getOffsetTotal()
     
     public function isComplet()
     {
-        return $this->getRows() == count($this->getContentRows());
+        return $this->getRows() == $this->_contentRowsCount;
```

Comment:
I would suggest to use the getter method `getContentRowCount` instead of accessing the privat property.